### DISCUSSION TITLE
Screen flash on launch (dark mode)

### DISCUSSION THREAD

**Discussion by wdavis** [spatie/ray#326] [General]
Does this bug anyone else?

https://user-images.githubusercontent.com/431631/109312732-ffb62580-780c-11eb-8b82-6ae36f76f1c9.mov

**Comment by AdrianMrn** [comment]
I'll have a look into this!

**Reply by AdrianMrn** [reply]
This will be fixed in 1.14.0!

**Reply by wdavis** [reply]
Awesome! Thank you
